src/main/scala/Server.scala

```diff
@@ -0,0 +1,19 @@
+import repo.Repository.InMemoryCustomerRepository
+import cats.effect._
+import endpoints.CustomerRoutes.customerRoutes
+import org.http4s.HttpApp
+import org.http4s.blaze.server.BlazeServerBuilder
+import org.http4s.server.Router
+import org.http4s.implicits._
+
+object Server extends IOApp {
+  private val customerRepo = new InMemoryCustomerRepository
+
```

Comment:
all the dependencies structure of the project

val repository1 = ...
val repository1 = ...
val repository1 = ...
val service1 = service1(repository1)
val service2 = service2(repository2, service1)
...

Reply:
and "layer" approach create is wrapped in `IO` that why need to be created in for-comprehension etc